recipe/meta.yaml

```diff
@@ -268,19 +297,24 @@ outputs:
       track_features:
         - hcc          # [variant=="hcc"]
       string: {{ variant }}_h{{ PKG_HASH }}_{{ build_number }}
+      ignore_run_exports_from:
+        # only contains binaries, so we can trust the link check
+        - zlib  # [unix]
```

Comment:
I don't understand this comment. Can you clarify?

Reply:
Ah, you mean the package only contains binaries.

Reply:
Yes, that's what I [meant](https://github.com/regro/libcfgraph/blob/master/artifacts/clang-15/conda-forge/linux-64/clang-15-15.0.7-default_had23c3d_1.json#L227). How should I describe it to be more less ambiguous?